modules/signing_worker/files/update-requirements.sh

```diff
@@ -1,11 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-if [ -z "$1" ]; then
-    echo "Usage: $0 worker_type"
-    echo "Worker type is ff-prod, tb-prod, or dep"
-    echo "Pinned requirements will be output to stdout"
-    exit 1
-fi
+# This script will create a docker image on the local machine with name "pip-compile"
+# It is safe to delete such image after running this script
+# It will also fetch yq and python images which can be safely removed from the local image cache
+
+# Configuration
+yq_docker_image=mikefarah/yq:4.28.2
```

Comment:
I sort of understand using docker to force the python version, but yq should presumably be just fine to keep running outside of docker?

Reply:
So I had an older version of `yq` on my system and it failed to parse the files ><, upgrading to 4.28.2 fixed it (unsure what the minimum version is tho).

Reply:
> I sort of understand using docker to force the python version, but yq should presumably be just fine to keep running outside of docker?

As ahal mentioned, there are minor differences with the versions, and in some systems `yq` installation (mac vs debian vs arch etc) can be a pain for the developer.
The `yq` docker image here is 18MB and very fast